test/src/hls-adapter.spec.js

```diff
@@ -483,6 +484,109 @@ describe('HlsAdapter Instance - change media', function() {
       });
     });
   });
+
+  it('should fire FRAG_LOADED', done => {
+    try {
+      hlsAdapterInstance = HlsAdapter.createAdapter(video, source1, config);
+      hlsAdapterInstance.addEventListener(EventType.FRAG_LOADED, event => {
+        event.payload.miliSeconds.should.exist;
+        event.payload.bytes.should.exist;
+        event.payload.url.should.not.be.empty;
```

Comment:
does it pass the test? URL doesn't exist on the change [previous change](https://github.com/kaltura/playkit-js-hls/commit/06a9f13556cd38f33a936660172553682d69761b#diff-b654f992dafabe8e34f424c597f47e28R1000)

Reply:
I don't understand the question. The change was done in FEC-8975 so the URL does exist now since 20/6/19. I think you did the wrong diff

Reply:
Look on the link that I added, did you add it on another fix?

Reply:
yes